qiime2/core/archive/provenance.py

```diff
@@ -269,10 +269,15 @@ def handle_metadata(self, name, value):
         else:
             raise NotImplementedError
 
+        _uuid = ""
```

Comment:
(Also usually you put the `_` on the end of the variable if you're trying not to clobber a builtin, but I think adding `ref` will make it clearer)